config/functions.config.local.js

```diff
@@ -4,6 +4,12 @@ var config = {
     },
     "sendgrid": {
         "key": process.env.SENDGRID_API_KEY
+    },
+    "verif_server": {
+        "url": "**url**",
+        "email": "**email**",
+        "password": process.env.VERIF_SERVER_PASSWORD,
+        "key": process.env.VERIF_SEVER_FIREBASE_API_KEY
```

Comment:
This isn't a secret key, so we can just include it directly in the config rather than make it an environment var.